src/main/webapp/scripts/script.js

```diff
@@ -8,3 +8,11 @@ function updateDom(text, containerId) {
 function getDomValue(domId) {
   return document.getElementById(domId).value;
 }
+
+function makeSelectList(value, text, selectId) {
```

Comment:
The function name "makeSelectList" seems to me that it returns an entire select element with all the options set up, while what it actually does is append one option to the select element. What do you think?